content/en/docs/cover-ui-assembly/_index.md

```diff
@@ -25,19 +25,43 @@ You need `.uf2` file from `firmware.zip`. Hold `BOOT` button on Pico, insert USB
 
 This is described in more details in [MicroPython guide](https://www.raspberrypi.com/documentation/microcontrollers/micropython.html#drag-and-drop-micropython), you can follow that, just use our `.uf2` file instead.
 
-## Meaning of LEDs and buttons
-
-Borrowed from Apehaenger's [custom firmware project](https://github.com/Apehaenger/CoverUI/tree/main/Firmware/CoverUI/yfc500#usage).
-
-* 2hr-8hr LEDs act as 4 digit progressbar for GPS quality. Blink = uncalibrated
-* Home button: Stop mowing and go back home to dock
-* Play button: Start mowing of recorded area(s)
-* S1: On = ROS is running but idle, Blink-slow = ROS in autonomous mode, Blink-fast = ROS in nirvana?
-* S2 LED: >TODO<
-* S2 Button: Short press, skip over to next mowing area. Long press (>= 3s), delete all recorded areas!
-* Lock Button: Clear emergency state (probably long press >=3s)
-* Mon-Sun as 7 digit progressbar for battery charge state (only in undocked state)
-* Lifted: Show emergency states. >TODO<
-* Wire: Also indicate a poor GPS. On = < 25%, Blink-fast = < 50%, Blink-slow < 75%, Off >= 75%
-* Battery: On = Battery empty
-* Charging LED: Fast blink = empty. Slow blink = approx. 1/2 charged. On = Fully charged.
+## Meaning of LEDs
+
+* **1st row** : 2hr-8hr act as 4 digit progressbar for GPS quality. Blink = uncalibrated.
```

Comment:
uncalibrated or "waiting for GPS fix" ?

Reply:
I just looked at the code, strictly speaking it blinks when gps quality is 0. It can happen in several cases:
- gps is just stating (when undocking)
- gps connection is bad (timeout), it's either the board or a network issue to connect to the NTRIP server
- gps quality is very low
- gps is not enabled or requested to work. The robot is in Idle mode but not in docking station, it can happen when an emergency blocked the robot for some time when mowing.

So overall, "uncalibrated" seems alright, as well as "gps quality is 0" but I'm not sure about "waiting for GPS fix" because in case 4. there is no waiting. 
Note that in this case, sometime it can be due to the wifi and there is no indication on the Cover UI. We should improve that.